src/datasources/memory.datasource.ts

```diff
@@ -0,0 +1,6 @@
+import { juggler, DataSourceConstructor } from '@loopback/repository';
+
+export const memds = new DataSourceConstructor({
```

Comment:
In LB 3.x, we are using `db` as the default datasource name. Can you do the same here please? Also the file name should become `db.datasource.ts` then.